components/CaseStatus/EditCaseStatusForm/EditCaseStatusForm.tsx

```diff
@@ -61,6 +75,50 @@ const EditCaseStatusForm: React.FC<{
     });
   }
 
+  if (caseStatusType == 'LAC' && (action == 'update' || action == 'end')) {
+    if (
+      currentCaseStatusStartDate &&
+      currentCaseStatusStartDate !== 'undefined'
+    ) {
+      const currentStartDateValidation = new Date(currentCaseStatusStartDate);
+
+      if (action == 'update') {
+        currentStartDateValidation.setDate(
+          currentStartDateValidation.getDate() + 1
+        );
+      }
+
+      form_fields.map((field: any) => {
+        if (field.id === 'startDate' || field.id === 'endDate') {
+          field.startDate = format(
+            new Date(currentStartDateValidation),
+            'yyyy-MM-dd'
+          );
+
+          field.default = format(
+            new Date(currentStartDateValidation),
+            'yyyy-MM-dd'
+          );
+        }
+      });
+    }
+  }
+
+  if (caseStatusType == 'LAC' && action == 'edit') {
+    if (pastCaseStatusStartDate && pastCaseStatusStartDate !== 'undefined') {
+      const caseStatusStartDate = new Date(pastCaseStatusStartDate);
+      caseStatusStartDate.setDate(caseStatusStartDate.getDate() + 1);
+
+      form_fields.map((field: any) => {
+        if (field.id === 'startDate') {
+          field.startDate = format(new Date(caseStatusStartDate), 'yyyy-MM-dd');
+
+          field.default = format(new Date(caseStatusStartDate), 'yyyy-MM-dd');
```

Comment:
I think it might make more sense to the user if the default was currentCaseStatusStartDate

Reply:
Good one!
Changed the default to +1 day